roles/common/templates/logrotate_rules.j2

```diff
@@ -0,0 +1,120 @@
+# {{ ansible_managed | comment }}
+# Logrotate configuration for {{ item.name }}
+
+{% for path in item.paths %}
+{{ path }}
+{% endfor %} {
+{% if item.options.daily is defined and item.options.daily %}
+    daily
+{% elif item.options.weekly is defined and item.options.weekly %}
+    weekly
+{% elif item.options.monthly is defined and item.options.monthly %}
+    monthly
+{% elif item.options.yearly is defined and item.options.yearly %}
+    yearly
+{% endif %}
+{% if item.options.rotate is defined %}
+    rotate {{ item.options.rotate }}
+{% endif %}
+{% if item.options.size is defined %}
+    size {{ item.options.size }}
+{% endif %}
```

Comment:
Here, too, we should standardize more. We want all logs to define a maximum size for rotation.